lsr_role2collection/collection_readme.txt

```diff
@@ -0,0 +1,36 @@
+﻿# Linux System Roles Ansible Collection
+Ansible collection for interacting with the Linux System Roles.
+
+## Currently supported distributions
+* Fedora
+* Red Hat Enterprise Linux (RHEL 6+)
+* RHEL 6+ derivatives such as CentOS 6+

```

Comment:
We should add something like "Some roles do not support RHEL6 and RHEL7.  See the documentation for the individual role you are interested in."

Reply:
+1. I suggest something like "NOTE: Some roles are not supported in RHEL6 and RHEL7. For more details about the individual roles you are interested in, see the documentation. "